From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 19 Jan 2022 19:13:56 +0100
Subject: [PATCH] Use system-provided docbook.xsl

Use docbook.xsl from docbook-xsl package instead of downloading it from
internet.

--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -7,10 +7,10 @@
         --stringparam man.copyright.section.enabled 0
 
 .xml.1:
-	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
+	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) /usr/share/xml/docbook/xsl-stylesheets-*/manpages/docbook.xsl $<
 
 .xml.5:
-	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
+	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) /usr/share/xml/docbook/xsl-stylesheets-*/manpages/docbook.xsl $<
 
 man_MANS = \
 	xdg-user-dir.1 \
